apps/portal/js/dashboard.js

```diff
@@ -243,24 +243,24 @@ $(function () {
      * @param page Current page
      * @returns {Array} List of allowe roles
      */
-    var gerUserAllowedViews = function (page) {
+    var getUserAllowedViews = function (page) {
         $('#ds-allowed-view-list').empty();
         var allowedViews = [];
         var pageViews = JSON.parse(JSON.stringify(page.layout.content));
         var viewKeysArray = Object.keys(pageViews);
-        var viewsArrayLength = viewKeysArray.length;
 
-        for (var i = 0; i < viewsArrayLength; i++) {
+        for (var i = 0, viewsArrayLength = viewKeysArray.length; i < viewsArrayLength; i++) {
```

Comment:
Fixed in commit 6f8fd741e62668ab01c395dc025ddcf9a5321de8
https://github.com/prabushi/carbon-dashboards/commit/6f8fd741e62668ab01c395dc025ddcf9a5321de8#diff-a86b0ec8b3aaab3d666a955e04a383faR250